src/trsp/GraphDefinition.cpp

```diff
@@ -640,11 +640,11 @@ bool GraphDefinition::addEdge(edge_t edgeIn) {
     if (itNodeMap != m_mapNodeId2Edge.end()) {
         // Connect current edge with existing edge with end node
         // connectEdge(
-        int64 lEdgeCount = itNodeMap->second.size();
+        int64 lEdgeCount = static_cast<int64>(itNodeMap->second.size());
```

Comment:
as in other cases, I feel that this is better declared with auto

Reply:
well, I did not made that code, I just wanted to make the minimal amount of changes.
that means just add `static_cast` every where on trsp code
actually is the function where more `static_cast` were added, and changing the declaration to auto will obscure the original intention of the original developer.
The problem on trsp goes more deeper. Like wrong choice if types for member variables on classes.
if you see the code, auto is almost never used.
I think changing trsp code to use `auto` would be when that code is rewritten, right now the objective is to have the least amount of warnings